clic/include/tier1/cleCropKernel.hpp

```diff
@@ -0,0 +1,52 @@
+
+#ifndef __TIER1_CLECROPKERNEL_HPP        
+#define __TIER1_CLECROPKERNEL_HPP        
+
+#include "cleOperation.hpp"                 
+
+namespace cle                                        
+{
+
+class CropKernel : public Operation     
```

Comment:
Missing methods to provide a second point for defining the crop area

Reply:
possible inspiration form [here](https://github.com/clEsperanto/pyclesperanto_prototype/blob/ea84c9fb11484d3682daee57316518e64ae0ffe8/pyclesperanto_prototype/_tier1/_crop.py#L8)